demobot.py

```diff
@@ -68,43 +68,43 @@ async def add(ctx:Context):
     if s not in q:
         q.append(s)
         logging.info('{0} add {1}'.format(ctx.guild,s))
-        await ctx.send('added you to the queue,\n' + getCustomAddMessage(ctx.guild))
+        await ctx.send(s + ' has been added to the queue. ' + getCustomAddMessage(ctx.guild))
```

Comment:
but adding one in here?

Reply:
yeah added one here so the new custom message appears after the original message